data/scripts/install-man-pages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/env sh
+
+# Set PREFIX.
+if [ -z  "${1+x}" ]; then
+    if ! [ -z "${PREFIX+x}" ]; then
```

Comment:
We should use `-n` instead of `! -z` for the sake of simplification and readability.

Reply:
Commendable motive. And I definitely don't mean to rock the boat. However, just want to point out that they're not perfect synonyms. When using the `[ ]` or the `test` builtins to evaluate expressions, in `sh`, `bash` and `zsh`, `[ -n ]` / `test -n`, that is `-n` with an empty string beside it so in this case with `${PREFIX}` empty, would evaluate to true, which usually is the opposite of what you want.

The effect would be an empty `${PREFIX}` output in case of an empty or undefined `${PREFIX}` input, while we wanted `/usr`.

This is also slightly related to why I prefer `${var+x}` instead of `$var`, it allows you to detect if the user or caller or whatever explicitly wanted an empty `var`, which would make no sense in the case of `PREFIX` or would it?! I guess `PREFIX=/` would be the same thing. But I find `+x` good practice alongside `set -u`.

I like the other suggestions, but maybe leave this one like so?

Reply:
You would be completely right if we didn't quote `${PREFIX+x}`. `-n` will evaluate to false when presented with an empty string, it will however evaluate to true when presented with, well, nothing. By quoting the variable we are creating a string, if the variable is empty the string will stay empty, but still a string. I think this can be better shown with examples:

`[ -n ]` evaluates to true
`[ -n "" ]` evaluates to false
`[ -n $test ]` evaluates to true when `$test` is empty
`[ -n "$test" ]` evaluates to false when `$test` is empty

We can drop the `+x` and have this being correctly evaluated with simpler code :grin:.